test/integration/heroku_alert_test.rb

```diff
@@ -0,0 +1,8 @@
+require_relative "../integration_test_helper"
+
+class HerokuAlertTest < ActionDispatch::IntegrationTest
+  should "not display the heroku alert if not on heroku" do
+    get "/"
+    assert !page.body.include?("PLEASE NOTE THAT THIS IS ONLY A TEST VERSION")
```

Comment:
I think it might be good to also have a test for the positive case as well, otherwise we don't know if the banner is working.